core/arch/aarchxx/mangle.c

```diff
@@ -74,12 +74,16 @@ typedef struct ALIGN_VAR(16) _icache_op_struct_t {
      * consecutive icache lines (which could wrap around the top of memory).
      */
     void *begin, *end;
+    /* cache the ctr el0 at init */
```

Comment:
style nit: UIse capitalization and punctuation in comments, esp whole-line comments.

Ditto at several places below.

Reply:
Done.